e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -200,23 +200,145 @@ test.describe('Example Imagery', () => {
+
+    // Click text=Example Imagery
+    await page.click('text=Example Imagery');
+
+    // Clear and set Image load delay (milliseconds)
+    await page.click('input[type="number"]', {clickCount: 3})
+    await page.type('input[type="number"]', "20")
+
+    // Click text=OK
+    await Promise.all([
+        page.waitForNavigation({waitUntil: 'networkidle'}),
+        page.click('text=OK'),
+        //Wait for Save Banner to appear
+        page.waitForSelector('.c-message-banner__message')
+    ]);
+    // Wait until Save Banner is gone
+    await page.waitForSelector('.c-message-banner__message', { state: 'detached'});
+    await expect(page.locator('.l-browse-bar__object-name')).toContainText('Unnamed Example Imagery');
+    const bgImageLocator = await page.locator(backgroundImageSelector);
+    await bgImageLocator.hover();
+    
+    // Click previous image button
+    const previousImageButton = await page.locator('.c-nav--prev');
+    await previousImageButton.click();
+
+    // Verify previous image 
+    const selectedImage = page.locator('.selected');
+    await expect(selectedImage).toBeVisible();
+
+    // Zoom in
+    const originalImageDimensions = await page.locator(backgroundImageSelector).boundingBox();
+    await bgImageLocator.hover();
+    const deltaYStep = 100; // equivalent to 1x zoom
+    await page.mouse.wheel(0, deltaYStep * 2);
+    const zoomedBoundingBox = await bgImageLocator.boundingBox();
+    const imageCenterX = zoomedBoundingBox.x + zoomedBoundingBox.width / 2;
+    const imageCenterY = zoomedBoundingBox.y + zoomedBoundingBox.height / 2;
+
+    // Wait for zoom animation to finish 
+    await bgImageLocator.hover();
+    const imageMouseZoomedIn = await page.locator(backgroundImageSelector).boundingBox();
+    expect(imageMouseZoomedIn.height).toBeGreaterThan(originalImageDimensions.height);
+    expect(imageMouseZoomedIn.width).toBeGreaterThan(originalImageDimensions.width);
+
+    // Center the mouse pointer
+    await page.mouse.move(imageCenterX, imageCenterY);
+
+    // Pan Imagery Hints
+    console.log('process.platform is '+process.platform);
+    const expectedAltText = process.platform === 'linux' ? 'Ctrl+Alt drag to pan' : 'Alt drag to pan';
+    const imageryHintsText = await page.locator('.c-imagery__hints').innerText();
+    expect(expectedAltText).toEqual(imageryHintsText);
+
+    // Click next image button
+    const nextImageButton = await page.locator('.c-nav--next');
+    await nextImageButton.click();
+
+    // Click fixed timespan button
+    await page.locator('.c-button__label >> text=Fixed Timespan').click();
+
+    // Click local clock 
+    await page.locator('.icon-clock >> text=Local Clock').click();
+
+    // Zoom in on next image
+    await bgImageLocator.hover();
+    await page.mouse.wheel(0, deltaYStep * 2);
+    const zoomedNextBoundingBox = await bgImageLocator.boundingBox();
+    const imageNextCenterX = zoomedNextBoundingBox.x + zoomedNextBoundingBox.width / 2;
```

Comment:
Variable is declared but never used, probably should be removed.

Reply:
Yes will do.